var/spack/repos/builtin/packages/hipsycl/package.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from os import path
+from llnl.util import filesystem
+
+
+class Hipsycl(CMakePackage):
+    """hipSYCL is an implementation of the SYCL standard programming model
+    over NVIDIA CUDA/AMD HIP"""
+
+    homepage = "https://github.com/illuhad/hipSYCL"
+    url = "https://github.com/illuhad/hipSYCL/archive/v0.8.0.tar.gz"
+    git = "https://github.com/illuhad/hipSYCL.git"
+
+    maintainers = ["nazavode"]
+
+    provides("sycl")
```

Comment:
@nazavode Since this is a new virtual spec, you need to provide a default in https://github.com/spack/spack/blob/develop/etc/spack/defaults/packages.yaml

Reply:
Yep, added default in b64e7829d6ce29621cffa543a6e0e5b909d5665b